js/components/tabledata/index.js

```diff
@@ -0,0 +1,49 @@
+import React, {Component} from 'react';
+import {Table, TableBody, TableHeader, TableHeaderColumn, TableRow, TableRowColumn} from 'material-ui/Table';
+import Diagram from '../diagram';
+
+export default class Tabledata extends Component {
+  constructor  (...args) {
+    super(...args);
+
+    this.state = {
```

Comment:
you must not save all this properties in state

Reply:
ok, i fix this